tests/tile/norm_fiber.cc

```diff
@@ -0,0 +1,79 @@
+/*! @copyright (c) 2022-present Skolkovo Institute of Science and Technology
+ *                              (Skoltech), Russia. All rights reserved.
+ *                 2023-present Artificial Intelligence Research Institute
+ *                              (AIRI), Russia. All rights reserved.
+ *
+ * NNTile is software framework for fast training of big neural networks on
+ * distributed-memory heterogeneous systems based on StarPU runtime system.
+ *
+ * @file tests/tile/norm_fiber.cc
+ * Euclidean norms over slices into a fiber of a Tile<T>
+ *
+ * @version 1.1.0
+ * */
+
+#include "nntile/tile/norm_fiber.hh"
+#include "nntile/starpu/norm_fiber.hh"
+#include "../testing.hh"
+#include <cmath>
+
+using namespace nntile;
+using namespace nntile::tile;
+
+template<typename T>
+void validate()
+{
+    using Y = typename T::repr_t;
+    const Y eps = T::epsilon();
+    // constants
+    Index batch = 5;
+    Index m = 3;
+    Index n = 20;
+    Index k = 1;
+    Scalar alpha = 1.0, beta = 0.0;
+    int redux = 0;
+    Index batch_ndim = 0;
+    int axis = 0;
+    // tiles
+    Tile<T> src1({batch, m, n, k});
+    Tile<T> src2({batch});
+    Tile<T> dst({batch});
+
+    // fill tiles
+    auto src1_local = src1.acquire(STARPU_W);
+    for(Index i = 0; i < src1.nelems; ++i)
+    {
+        src1_local[i] = Y(-1.0);
+    }
+    src1_local.release();
+
+    auto src2_local = src2.acquire(STARPU_W);
+    for(Index i = 0; i < src2.nelems; ++i)
+    {
+        src2_local[i] = Y(1.0);
+    }
+    src2_local.release();
```

Comment:
dst is not initialized with some non-zero values here. Zeros may hide implementation bugs.